extensions/machine-learning/src/views/models/tableSelectionComponent.ts

```diff
@@ -17,7 +17,8 @@ export interface ITableSelectionSettings {
 	databaseTitle: string,
 	tableTitle: string,
 	databaseInfo: string,
-	tableInfo: string
+	tableInfo: string,
+	layout?: string,
```

Comment:
can you please make sure the page in predict wizard which using this component doesn't break?

Reply:
@llali I checked and the layout behaves the same in predict wizard.